src/main/java/org/eclipse/tractusx/ssi/lib/validation/JsonLdValidatorImpl.java

```diff
@@ -31,7 +31,7 @@
 import jakarta.json.JsonObject;
 import jakarta.json.JsonValue;
 import java.util.Map;
-import org.eclipse.tractusx.ssi.lib.exception.InvalidJsonLdException;
+import org.eclipse.tractusx.ssi.lib.exception.json.InvalidJsonLdException;
```

Comment:
Throws **RuntimeException** (see line 98) when an undefined term was found, but it should rather throw an **InvalidJsonLdException**